### ISSUE TITLE
Web - Hmm... it's not here page displayed after clicking on the card transaction preview

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#32155]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___
Issue found when executing #31353 
**Version Number:** 1.4.4-3
**Reproducible in staging?:** y
**Reproducible in production?:** not able to check
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** applause internal team
**Slack conversation:**

## Action Performed:
Pre-requisite: an account with the expensify card and transactions on that card
1. In OldDot, login into the account
2. Navigate to the report that contains the Expensify card transactions
3. Copy the reportID shown in the URL
4. In NewDot, navigate to /r/<reportID> where reportID is the reportID copied in step 3
5. Click on the card transaction preview

## Expected Result:
The transaction preview is displayed

## Actual Result:
Hmm... it's not here page displayed after clicking on the card transaction preview

## Workaround:
unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [x] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

https://github.com/Expensify/App/assets/43996225/f6ae8392-e24f-4773-b2ce-74aa9c845795

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by junsupark94** [comment]
Contributor details
Your Expensify account email: junsupark94@gmail.com
Upwork Profile Link: https://www.upwork.com/freelancers/~01f0918c3cf48c22cd

**Comment by grgia** [comment]
@luacmartins do you happen to know who worked on adding cash transactions from OldDot to NewDot? It seems like we're fetching a bunch of cash transactions created outside of NewDot that never went through the money request flow.

**Comment by luacmartins** [comment]
Do you mean card transactions? From the OP this seems like a backwards compatibility issue, since it doesn't seem like we fetched those, we are manually navigating to a report we can only see in OldDot. As far as the migration for backwards compatibility goes, we deemed it out of the scope in the [design doc](https://docs.google.com/document/d/1vnZSy0ot3Ykz7sRTw2A5Cvuj9_azjkj90ikE_iuwvF8/edit#bookmark=id.viux8o15e84k) so this is not done, but maybe now is the time to do it?

>We also have some backwards compatibility concerns here, since existing company card transactions don’t have a corresponding Workspace chat, ReportPreview action, IOU reportAction and existing company card transactions would need to be migrated. However, this was [considered out of scope for Money 20/20](https://expensify.slack.com/archives/C027726KHT4/p1694092081146929) deadline and would be tackled as a phase 2 part of the project.

**Comment by garrettmknight** [comment]
@grgia do you think this is worth prioritizing now?

**Comment by grgia** [comment]
@garrettmknight TBD, I think if this pertains to old reports only, then no. But if it's a problem with our core transaction creation logic then yes- though we should try to do specific internal testing of this flow with real card transactions and reports before we work on this bug

**Comment by luacmartins** [comment]
>@luacmartins but if the transaction exists in NewDot, then during the create transaction flow we should be creating the report in NewDot too shouldn't we?

Yes, I read the OP wrong 😓 So the transactions do show up in NewDot but it seems like we have an issue opening the transaction thread. I couldn't reproduce the issue locally and the logs for reportID 2966681377842678 used in the video don't show any errors.

@kavimuru could you please retest this and provide a video?

**Comment by kavimuru** [comment]
> @kavimuru could you please retest this and provide a video?

@luacmartins Sure will do!

**Comment by luacmartins** [comment]
Cool, I think we're good to close this then.
